ludwig/hyperopt/execution.py

```diff
@@ -698,9 +694,20 @@ def run_experiment_trial(config, local_hyperopt_dict, checkpoint_dir=None):
         run_experiment_trial_params = tune.with_parameters(run_experiment_trial, local_hyperopt_dict=hyperopt_dict)
         register_trainable(f"trainable_func_f{hash_dict(config).decode('ascii')}", run_experiment_trial_params)
 
+        class CallbackStopper(Stopper):
```

Comment:
Could we move this class declaration outside, since this function is already very long?

i.e.


```python
class CallbackStopper(Stopper):
     """Blanket callback for whether to stop hyperopt."""

     def __init__(self, callbacks: List[Callback]):
         self.callbacks = callbacks

     def __call__(self, trial_id, result):
                return False

            def stop_all(self):
                for callback in self.callbacks or []:
                    if callback.should_stop_hyperopt():
                        return True
                return False

"""
inside execute()
"""

callback_stopper = CallbackStopper(callbacks)

analysis = tune.run(
    ...
    stop=callback_stopper,
    ...
)
```

Reply:
Done.